gem/chhaajja/arresunna/migrations/0003_auto_20190511_1204.py

```diff
@@ -0,0 +1,23 @@
+# Generated by Django 2.2.1 on 2019-05-11 06:34
+
+from django.conf import settings
+from django.db import migrations
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('wagtailcore', '0041_group_collection_permissions_verbose_name_plural'),
+        ('wagtailredirects', '0006_redirect_increase_max_length'),
+        ('contenttypes', '0002_remove_content_type_name'),
+        ('wagtailforms', '0003_capitalizeverbose'),
+        migrations.swappable_dependency(settings.AUTH_USER_MODEL),
+        ('arresunna', '0002_auto_20190511_1114'),
+    ]
+
+    operations = [
+        migrations.RenameModel(
```

Comment:
since this PR has not yet landed, we don't need a migration that renames a model, we can delete the previous migration where the model was created and run makemigrations again so that there is one migration instead of two